resources/manage.rb

```diff
@@ -20,6 +20,7 @@
 def initialize(*args)
   super
   @action = :create
+  @sensitive = true
```

Comment:
IMHO no need to have this as a resource attribute. We can just set it true directly.